scripts/generateRules.mjs

```diff
@@ -0,0 +1,219 @@
+#!/usr/bin/env node
+
+import { readFile, writeFile } from 'fs/promises'
+import path from 'path'
+
+import 'dotenv/config'
+import chalk from 'chalk'
+import XLSX from 'xlsx'
+import lodash from 'lodash'
+
+import { DATA_SHEET_TYPES } from '../src/server/services/records.js'
+import { SERVER_DATA_DIR, EMPTY_TEMPLATE_NAME, SRC_DIR } from '../src/server/environment.js'
```

Comment:
I guess we don't have to solve this now, but have you thought about how we might support custom templates per reporting period?

Reply:
i really hope we don't have to support custom sets of rules per reporting period, though the cares reporter code definitely had some provisions for doing that